### PR TITLE
Fix USE_FPV

### PR DESCRIPTION
**Pull request by sctse999**
1. Fallback USE_FPV to use 'cam/image_array' as inf_input is normalized.
2. The video stream should fit the screen by height instead of width

Note: By using 'cam/image_array' , crop (if set) is not shown in the FPV. Just aim to fix the FPV at the moment as it is a bit tricky to incorporate the crop in the current codebase. Welcome any feedback. Thanks.

### CODE REVIEW

File: donkeycar/parts/web_controller/templates/base_fpv.html [hunk 1]
```diff
@@ -45,23 +44,33 @@
...
-    </div>
+    <img class="logo" src="/static/donkeycar-logo-sideways.png" alt="Donkeycar Logo">
 </div>
 
 <body style="background-color:rgb(48, 49, 51);">
 
 <div class="gap"></div>
-<img id='mpeg-image', class='img-responsive' src="/video"/> </img>
+<div>    
+    <img id='mpeg-image', class='img-responsive' src="/video"/> </img>
```
Comment (nvtkaszpir) [inline donkeycar/parts/web_controller/templates/base_fpv.html]: I believe that comma in here is not needed.

**Comment by sctse999** [comment]
This fix does not support cropping. It just fallback to use the original image captured by the camera. 

Before this commit, the FPV use ```inf_input``` as the input which is a normalized image that is not usable as a mjpeg.  I revert this by using the original image, which is the ```cam/image_array```.

If we want to support cropping, we might need to decouple the normalize and cropping operation in ```normalize_and_crop``` in [utils.py](https://github.com/autorope/donkeycar/blob/77b1dc76c03657e887428a99929c14f78bbb865f/donkeycar/utils.py#L133)

File: donkeycar/templates/complete.py [hunk 2]
```diff
@@ -301,7 +301,7 @@ def run(self, img_arr):
 
     # Use the FPV preview, which will show the cropped image output, or the full frame.
     if cfg.USE_FPV:
-        V.add(WebFpv(), inputs=[inf_input], threaded=True)
+        V.add(WebFpv(), inputs=['cam/image_array'], threaded=True)
```
Comment (DocGarbanzo) [inline donkeycar/templates/complete.py]: I thought that was what I had in my original PR, maybe it got overwritten later.

File: donkeycar/parts/web_controller/templates/base_fpv.html [hunk 3]
```diff
@@ -45,23 +44,33 @@
...
+    <img class="logo" src="/static/donkeycar-logo-sideways.png" alt="Donkeycar Logo">
 </div>
 
 <body style="background-color:rgb(48, 49, 51);">
 
 <div class="gap"></div>
-<img id='mpeg-image', class='img-responsive' src="/video"/> </img>
+<div>    
+    <img id='mpeg-image' class='img-responsive' src="/video"/> </img>
+</div>
```
Comment (DocGarbanzo) [inline donkeycar/parts/web_controller/templates/base_fpv.html]: I checked your changes and they work well. Really better to fit by height than by width!

Comment (DocGarbanzo) [reply donkeycar/parts/web_controller/templates/base_fpv.html:73]: @tawnkramer - I think you can merge this straight away.

**Pull request merged**